upnp.py

```diff
@@ -76,10 +83,11 @@ def sploit(host):
  print "Trying to get some info from the target...";
  try:
   res = urllib2.urlopen(host).read();
+  print "Connected"
   res = res.replace("\r", "");
   res = res.replace("\n", "");
   res = res.replace("\t", "");
-  pres = res.split("<serviceId>urn:upnp-org:serviceId:WANIPConn1</serviceId>");
+  pres = res.split("<serviceId>urn:dslforum-org:serviceId:WANIPConn1</serviceId>");
```

Comment:
Might be better to try both or make it configurable. While you're adding support for your devices, it probably is breaking support for the others.